### PR TITLE
Rename autocomplete & remove validation related options

### PR DESCRIPTION
**Pull request by dylanBetty**

### CODE REVIEW

File: src/prefabs/structures/AutocompleteInput/options/validation.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { showOn } from '../../../../utils';
+
+export const validation = {
+  required: toggle('Required'),
+  validationValueMissing: variable('Value required message', {
+    value: ['This field is required'],
+    ...showOn('required'),
+  }),
+  closeOnSelect: toggle('Close dropdown after select', { value: true }),
+  disabled: toggle('Disabled'),
```
Comment (benjamin-rocks) [inline src/prefabs/structures/AutocompleteInput/options/validation.ts:12]: toggles should have a valid value, does this show up in the lint?

Comment (dylanBetty) [reply src/prefabs/structures/AutocompleteInput/options/validation.ts:12]: I've added an explicit value, it looks like the `attrs` object on the toggle function is optional 
```typescript
export declare const toggle: (label: string, attrs?: Attributes) => OptionProducer;
```

Comment (dylanBetty) [reply src/prefabs/structures/AutocompleteInput/options/validation.ts:12]: Ah one moment, I see disabled has been put down twice, my linter didn't catch that for some reason.

Comment (benjamin-rocks) [reply src/prefabs/structures/AutocompleteInput/options/validation.ts:12]: > ```export declare const toggle: (label: string, attrs?: Attributes) => OptionProducer;```

I thought i'd made it explicit, my mistake.

**Review by benjamin-rocks — APPROVED** [review]

**Pull request merged**